SPECS/telegraf/telegraf.spec

```
@@ -81,6 +85,9 @@ fi
 %dir %{_sysconfdir}/%{name}/telegraf.d
 
 %changelog
+* Thu Mar 20 2025 Mayank Singh <mayansingh@microsoft.com> - 1.31.0-6
+- Fix CVE-2024-35255 and CVE-2025-27144.patch with an upstream patch
```

```suggestion
- Fix CVE-2024-35255 and CVE-2025-27144.patch with an upstream patch
```
- Fix CVE-2024-35255 and CVE-2025-27144 with an upstream patch